webpack.config.js

```diff
@@ -8,11 +8,22 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const AngularCompilerPlugin = require('@ngtools/webpack').AngularCompilerPlugin;
 const pjson = require('./package.json');
+const config = require('./config.js')
 
 if (process.env.NODE_ENV == null) {
     process.env.NODE_ENV = 'development';
 }
-const ENV = process.env.ENV = process.env.NODE_ENV;
+
+const NODE_ENV = process.env.NODE_ENV;
+const envConfig = config.load(process.env.ENV)
+config.log(envConfig)
+
+if (process.env.ENV == null) {
+    // Defaults to development if an ENV is not explicitly chosen. Also preserves the selfHost option
+    process.env.ENV = 'development' 
```

Comment:
This seems to run after `config.load`. Seems reasonable to put it before?

Reply:
After chatting, I've changed the logic to assume that there is not an initial `development.json` override file. The default values (stored in `base.json`) happen to be the development values. I was thinking to provide a default experience without requiring the setup step of creating `development.json` previous to being able to run the project, but allowing the flexibility to override the default experience with the `development.json`

This placement allows us to preserve the choice (or the lack of) while running webpack to load the config files

### Current cases:
#### [ `base.json` ], `cross-env NODE_ENV=development webpack`
all default values that we provide. This is the default if no ENV is passed through. Most likely, `development.json` doesn't exist in this case.

#### [ `base.json`, `development.json` ], `cross-env NODE_ENV=development ENV=development webpack`
override some/all default dev values provided by `base` with what is in the user created `development.json`

#### [ `base.json`, `qa.json` ], `cross-env NODE_ENV=production ENV=qa webpack`
overrides the default dev values to run against the QA environment. 

#### [ `base.json`, `prod.json` ], `cross-env NODE_ENV=production ENV=production webpack`
overrides the default values to run against the production environment.